test/DebugInfo/NonSemantic/Shader200/FortranArrayNoType.spt

```diff
@@ -0,0 +1,73 @@
+; RUN: llvm-spirv -to-binary %s -o %t.spv
```

Comment:
Add intro comment describing the purpose of this test, similar to the other testcases.

Reply:
Applied